src/theme/common.less

```diff
@@ -20,7 +20,7 @@
  */
 
 
- @mainColor: #040060;
+ @mainColor: #000000;
```

Comment:
I just noticed that changing this will also force the background-color in src/theme/styles.less as well.

We'll need that value to be white to ensure it matches the header. I've attached a screenshot of what it looks like presently:
![image](https://user-images.githubusercontent.com/39800563/72360267-b31d9280-36bd-11ea-9890-f7df204a267a.png)